src/commands/mod.rs

```diff
@@ -664,11 +728,11 @@ mod tests {
 
         let expected_contents = r#"---
 app_id: test_app
-user_version: ''
-description: ''
-icon: ''
-author: ''
-homepage_url: ''
+user_version: null
+description: null
```

Comment:
What is this ? We don't support null. We only support strings... or ut should not be in the manifest file

Reply:
So if we have EdgeAppManifest like this (default for optional struct fields):

```       
 let manifest = EdgeAppManifest {
            app_id: "test_app".to_string(),
            settings: vec![Setting {
                title: "username".to_string(),
                type_: SettingType::String,
                default_value: "stranger".to_string(),
                optional: true,
                help_text: "An example of a setting that is used in index.html".to_string(),
            }],
            ..Default::default()
        };
```
It should not write this fields to the `screenly.yml`?

Reply:
> So if we have EdgeAppManifest like this (default for optional struct fields):
> 
> ```
>  let manifest = EdgeAppManifest {
>             app_id: "test_app".to_string(),
>             settings: vec![Setting {
>                 title: "username".to_string(),
>                 type_: SettingType::String,
>                 default_value: "stranger".to_string(),
>                 optional: true,
>                 help_text: "An example of a setting that is used in index.html".to_string(),
>             }],
>             ..Default::default()
>         };
> ```
> 
> It should not write this fields to the `screenly.yml`?

If they are empty it should not.

Reply:
Update after daily: not going to check that those Optional fields are not empty (except for app_id -> app_id should still be checked for length)